lib/include/openamp/rpmsg.h

```diff
@@ -344,8 +343,8 @@ void rpmsg_destroy_ept(struct rpmsg_endpoint *ept);
  */
 static inline unsigned int is_rpmsg_ept_ready(struct rpmsg_endpoint *ept)
 {
-	return (ept->dest_addr != RPMSG_ADDR_ANY &&
-		ept->addr != RPMSG_ADDR_ANY);
+	return ept->dest_addr != RPMSG_ADDR_ANY &&
+		ept->addr != RPMSG_ADDR_ANY;
```

Comment:
better to be (ept->dest_addr != RPMSG_ADDR_ANY) &&
		(ept->addr != RPMSG_ADDR_ANY) for easier to read

Reply:
Ok